src/stopwords.c

```diff
@@ -188,3 +189,28 @@ void ReplyWithStopWordsList(RedisModuleCtx *ctx, struct StopWordList *sl) {
   RedisModule_ReplySetArrayLength(ctx, i);
   TrieMapIterator_Free(it);
 }
+
+char **GetStopWordsList(struct StopWordList *sl) {
+  size_t size = sl->m->cardinality;
```

Comment:
too many non-null assumptions here.

Reply:
`m`, the Trie that keeps the list is never NULL. 
We can assert but it feels like overkill to me.
The list itself is never NULL either as we have `__empty_stopwords` global list.